SignalShareExtension/SignalShareExtension-Bridging-Header.h

```diff
@@ -2,16 +2,24 @@
 //  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
 //
 
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+// Separate iOS Frameworks from other imports.
 #import "DebugLogger.h"
+#import "Environment.h"
 #import "OWSLogger.h"
+#import "OWSMath.h"
+#import "OWSPreferences.h"
```

Comment:
It might be simpler to just `#import <SignalMessaging/SignalMessaging.h>` in the bridging headers.